ts/nni_manager/training_service/local/localTrainingService.ts

```diff
@@ -170,18 +170,20 @@ class LocalTrainingService implements TrainingService {
         return trialJob;
     }
 
-    public async getTrialLog(trialJobId: string, logType: LogType): Promise<string> {
-        let logPath: string;
-        if (logType === 'TRIAL_LOG') {
-            logPath = path.join(this.rootDir, 'trials', trialJobId, 'trial.log');
-        } else if (logType === 'TRIAL_STDOUT'){
-            logPath = path.join(this.rootDir, 'trials', trialJobId, 'stdout');
-        } else if (logType === 'TRIAL_ERROR') {
-            logPath = path.join(this.rootDir, 'trials', trialJobId, 'stderr');
-        } else {
-            throw new Error('unexpected log type');
+    public async getTrialFile(trialJobId: string, fileName: string): Promise<string | Buffer> {
+        // check filename here for security
+        if (!['trial.log', 'stderr', 'model.onnx', 'stdout'].includes(fileName)) {
+            throw new Error(`File unaccessible: ${fileName}`);
+        }
+        let encoding: string | null = null;
+        if (!fileName.includes('.') || fileName.match(/.*\.(txt|log)/g)) {
```

Comment:
I suggest rename `stdout` to `stdout.txt`...
Not the responsibility of this pr though...